### ISSUE TITLE
No sync until verbose logging enabled after URI changed.

### ISSUE THREAD

**Issue by CupricReki** [vrtmrz/obsidian-livesync#363]
## Reproducing procedure

1. Set sync settings by preset 'Disable all automatic'
2. Change URI
3. Set Sync Settings by preset 'LiveSync' - No changes are synced 
5. Enable 'Verbose Logging' - Changes immediately sync

### Other information, insights and intuition.
I appreciate the effort that went into this plugin but the setup is really difficult and buggy. After about 6 hours of trial and error, multiple clean wipes, I finally have all my devices syncing. 

Again though. I thoroughly appreciate your work.

**Comment by vrtmrz** [comment]
Sorry for being very late!
In these cases, Obsidian command `Reload app without saving`  will help us! Please try if you are troubled next time!
I will check the reflection of the settings.

> really difficult

Yes, I am always wondering how can we improve this. As you mentioned, CouchDB is a bit minor database. I refined [the setup guide](https://github.com/vrtmrz/obsidian-livesync/blob/main/docs/setup_flyio.md) and [utilities](https://github.com/vrtmrz/obsidian-livesync/tree/main/utils) recently. 
I hope that the new guide will help you in the next time.

**Comment by CupricReki** [comment]
Yes, I could see how `Reload app without saving` may have worked in my situation. What about adding a notification when you close the setting when changes have been made which require a restart?

**Comment by CupricReki** [comment]
Difficulty of setup I believe can be attributed to a GUI designed for power users. Most users just want sync to work with minimal effort and good defaults. 

Present only necessary options by default and hide advanced options behind an `advanced options` checkbox.

For example:
### Remote Database Configuration
#### Confidentiality 
- End to End encryption by default.  Generate the encryption passphrase on initial setup.
- Always path obfuscate or never path obfuscate. 
- Remove Use dynamic iteration count. _If this is something the user benefits from, just enable it. If not, remove it_

**Comment by vrtmrz** [comment]
Thank you for your advice! Perhaps the biggest problem is that `Setup Wizard` is buried and not well known. When we start the `Setup Wizard`, we will see items only we must configure.

At v0.22.6, the `Setup Wizard` is refined. Brand new installed Self-hosted LiveSync prompts the dialogue to guide setup.  We can choose the setup method -- Using `Setup URI` or starting `Minimal setup` (Which has been refined from the `Setup wizard`).

And, the documents are also refined. Would you mind check the latest version and documents?
